blocks/columns/columns.css

```diff
@@ -19,15 +15,93 @@
   display: block;
 }
 
+/*
+ * Column Manufacturing Steps
+*/
+
+.section.columns-container h2 {
+  font-size: 24px;
+}
+
+.columns.block h2, .columns.block h3 {
+  font-size: 24px;
+}
+
+.columns.block > div > div {
+  order: 1;
+}
+
+.columns.block.icon-steps {
+  font-size: var(--body-font-size-xs);
+  display: grid;
+  word-wrap: break-word;
+}
+
+.columns.block.icon-steps > div {
+  background-color: #FFF;
+  box-shadow: 0 4px 10px 0 rgba(0 0 0 / 20%);
+  border-radius: 10px;
+  padding: 20px 30px;
+  margin: 0;
+  border: 0;
+}
+
+.columns.block.icon-steps > div:last-child {
+  background-color: #FFF;
```

Comment:
[Semantic names](https://github.com/hlxsites/takeda-ihs/blob/7c85cf7cc5528bc8a8a4e82d56180e8db6617c0c/styles/styles.css#L17-L22) instead of hex values. in this case the literal `white` is fine as that is equivalent.